database/seeders/BadakhshanProvinceSeeder.php

```diff
@@ -17,8 +17,36 @@ public function run():void
 
         // Create districts
         $province->districts()->createMany([
-            // Add districts here...
-            // ['name' => 'نام', 'en_name' => 'Name'],
+            [ 'name' => "( مرکز ولایت بدخشان) فیض آباد", "en_name' => "( مرکز ولایت بدخشان) فیض آباد" ],
```

Comment:
@hoshmandaqil The `en_name` is missing, it somehow the duplicate of Dari version.

Reply:
Fixed @hoshmandaqil